### PR TITLE
Basic server

### PR DESCRIPTION
**Pull request by laptou**
- Modify 'Hello World' server to say 'Hello Ibiyemi'

### CODE REVIEW

**Comment by jpnickolas** [comment]
For features like this that are orthogonal to previous PR's, you should try to branch from main instead of the most recent PR. This will help Jess and I review as well as save you some headache with merging.

**Comment by laptou** [comment]
> For features like this that are orthogonal to previous PR's, you should try to branch from main instead of the most recent PR. This will help Jess and I review as well as save you some headache with merging.

I will try to do this in the future, but I don't know how to update this existing branch to reflect that. Since this is downstream from master, rebasing doesn't seem to work.

**Comment by jpnickolas** [comment]
> > For features like this that are orthogonal to previous PR's, you should try to branch from main instead of the most recent PR. This will help Jess and I review as well as save you some headache with merging.
> 
> I will try to do this in the future, but I don't know how to update this existing branch to reflect that.

No worries. I'll probs hold off on reviewing this in detail until some of the larger parent PR's are merged.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (jpnickolas) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: Add a line to your website that fetches this data and inserts it somewhere. Even if it's just in an alert somewhere.

**Review by jpnickolas — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java
Comment (laptou) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:30]: The tutorial said to stop here and make a pull request, I was planning to make the version that inserts this into the body PR #16 .

**Review by jessicaslaughter — APPROVED** [review]

**Review by jpnickolas — APPROVED** [review]

**Pull request merged**
